Firmware/PlatformIO/main_mcu/src/bms_telemetry.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "teensy_can.h"
+#include "virtualTimer.h"
+
+const uint16_t kSignalsPerMessage = 7;
+const uint16_t kNumVoltageMessages = 20;
+const uint16_t kNumTemperatureMessages = 16;
+const int kHPTransmitPeriod = 10;
+const int kVBTransmitPeriod = 10;
+const int kLPTransmitPeriod = 100;
```

Comment:
These should be private member variables (you probably have to make them static or the compiler won't like how they're used)